Tools/Scripts/webkitpy/port/driver.py

```diff
@@ -359,8 +359,12 @@ def is_web_platform_test(self, test_name):
         return test_name.startswith(self.web_platform_test_server_doc_root)
 
     def wpt_test_path_to_uri(self, path):
-        should_use_https = ".https." in path or ".serviceworker." in path or ".serviceworker-module." in path
-        return self.web_platform_test_server_base_https_url + path if should_use_https else self.web_platform_test_server_base_http_url + path
+        if ".h2" in path:
```

Comment:
```suggestion
        if ".h2." in path:
```

to be consistent with the others, I think?

Reply:
Yeah this is a definitely a typo on my part.